bskyweb/cmd/bskyweb/server.go

```diff
@@ -168,6 +168,7 @@ func serve(cctx *cli.Context) error {
 	e.GET("/moderation/blocked-accounts", server.WebGeneric)
 	e.GET("/settings", server.WebGeneric)
 	e.GET("/settings/app-passwords", server.WebGeneric)
+	e.GET("/settings/preferences-home-feed", server.WebGeneric)
```

Comment:
Can we tune this path up a bit? maybe `/settings/home-feed` or at least `/settings/home-feed-preferences`

Reply:
I tried to keep the naming consistent with what is was before but I can change it

Reply:
Done